app/views/stories/_form_errors.html.erb

```diff
@@ -1,24 +1,25 @@
 <div class="form_errors_header">
-  <% if story.errors.count == 1 && story.already_posted_story %>
+  <% if story.errors.count == 1 && !story.already_posted_stories.empty? %>
     <div class="flash-error">
-    <h2>Error: This story was already submitted <%=
-      time_ago_in_words_label(story.already_posted_story.created_at) %></h2>
+    <h2>Error: This story was most recently submitted <%=
```

Comment:
```suggestion
    <h2>Error: This story was submitted <%=
```

This is more accuracy than the submitter needs to get what's going on, let's just drop it.